Tests/SwerverTests/ResponseSpec/HttpResponseSpec.swift

```diff
@@ -5,48 +5,104 @@ import Foundation
 
 class HttpResponseSpec: QuickSpec {
     override func spec() {
-        describe("HttpResponse") {
-            var responseBuilder = ResponseBuilder()
+        describe("A Reponse object") {
+            func summonBuilder() -> HttpResponseBuilder {
+                return HttpResponseBuilder()
+            }
             
-            it("can equate two response object") {
-                let responseOne = responseBuilder
-                    .with(statusCode: .ok)
-                    .with(contentType: .text)
-                    .with(body: String())
-                    .build()
-                
-                let responseTwo = responseBuilder
-                    .with(statusCode: .ok)
-                    .with(contentType: .text)
-                    .with(body: String())
-                    .build()
-                
-                expect(responseOne).to(equal(responseTwo))
+            context("when compares with another response object") {
+                it("is true when attributes match") {
+                    var responseBuilder = summonBuilder()
+                    
```

Comment:
I personally would not add a newline here, but that's just my style preference. To me, the line above the newline and the section below the newline are both part of the setup of the individual test.

Reply:
will take a look

Reply:
fixed